src/app/(safespace)/space/passwort/strength/page.tsx

```diff
@@ -104,37 +129,14 @@ export default function PasswordStrength() {
     rustle(clen * increment + 1);
   };
 
-  const loadHighscore = async () => {
-    const user = await userStore.loadUser();
-    return user ? user.highscores[Highscore.PASSWORD_STRENGTH] : 0;
-  };
-
-  const saveHighscore = async () => {
-    const user = await userStore.loadUser();
-    const highscoreEntry = user
-      ? user.highscores[Highscore.PASSWORD_STRENGTH]
-      : 0;
-    if (currentScore > highscoreEntry) {
-      userStore.setHighscore(Highscore.PASSWORD_STRENGTH, currentScore + 1);
+  const saveHighScore = async () => {
+    if (user !== null) {
+      const context = new PersistUserService();
+      return await context.setHighScore("PASSWORD_STRENGTH", currentScore);
```

Comment:
return await context.setHighScore("PASSWORD_STRENGTH", currentScore + 1);
+ this should NOT create a new entry in the database - it should update the existing valu - there is only ONE highscore entry for each user and game